### DISCUSSION TITLE
The panel should have other colors

### DISCUSSION THREAD

**Discussion by kindusmithkindu** [linuxmint/discussions#542] [Ideas]
Currently, cinnamon's panels are only available in white, black, and gray fixed colors. Why can't you add some panel theme and color choices, such as pink, blue, etc., instead of just a few fixed colors. In addition, I hope the system should add some panel layout themes, such as menu centering, instead of just the classic layout like xp.

**Comment by Jopp-gh** [comment]
Custom themes (in css) should be able to modify the bg col of cinnamon's panel.
Just go to  `settings>themes` and click the Advanced button.

Also, there's an plugin to make panels transparent, so check `settings>extensions>download`

Menu centering? You can move all applets / apps in your panels, as you like. Also, you can add more panels. Just right click on the panel, toggle edit etc

**Reply by rtcunningham1** [reply]
The Cinnamon Panel Tweaker applet will let you set any color for a specific panel.
